remo/dashboard/templates/dashboard_reps.jinja

```diff
@@ -1052,8 +1052,7 @@
 {% endif %}
 <!-- end planning requests block -->
 
-<!-- Admin block -->
-{% if is_admin is defined %}
+<!-- Strategy block -->
```

Comment:
By removing the `if` completely, all the users will be able to see an empty `Administration` heading. Probably you need to augment the if statement by adding all the users that should have access to it.

Reply:
Oh, you're completely right, missed that. Will do